app/controllers/authenticate_controller.rb

```diff
@@ -53,14 +53,14 @@ def update_config
       username: ::Role.username_from_roleid(current_user.role_id),
       enabled: body_params['enabled'] || false
     )
-    log_audit_event(update_authn_config_audit_event)
+    log_audit_event(update_config_audit_event)
```

Comment:
i think that if we do this here then we should do this for all the calls from the `authenticate_controller` (`validate_status`, etc.)